prac_05/wimbledon.py

```diff
@@ -0,0 +1,54 @@
+"""
+Word Occurrences
+Estimate: 120 mins
+Actual: roughly 2 or more hours split up over a few days
+"""
+filename = 'wimbledon.csv'
+
+
+def main():
+    """
+    Displays the champions and how many times they have won.
+    Display the countries of the champions in alphabetical order
+    """
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        # read first line in file to skip and get rest of lines
+        in_file.readline()
+        lines = [line.strip().split(',') for line in in_file]
+
+        champion_to_win_count = calculate_champion_to_win_count(lines)
+        display_champion_to_win_count(champion_to_win_count)
+
+        display_champion_countries(lines)
```

Comment:
Should the file remain open after reading the file?

Reply:
More clarification, should you be doing all this extra work inside the `with` block?

Reply:
I see what you're saying. My functions don't need to be in line with the with statement so I've fixed this.